frontend/src/registration/components/CongratulationsPage/CongratulationsPage.tsx

```diff
@@ -10,12 +10,15 @@ import { submitRegistration } from '../../actions';
 import { RegistrationPage } from '../RegistrationPage';
 import messages from './displayMessages';
 import './styles.scss';
+import { MatomoUserIdTracker } from '../../../utils/MatomoTracker';
```

Comment:
Nit: you don't need the relative imports here, you can use it like this: https://github.com/open-craft/opencraft/blob/99fc7777ae658e9610ff52f8c0aa4395952b54e6/frontend/src/console/components/CustomizationSideMenu/CustomizationSideMenu.tsx#L7